### PR TITLE
feat(cli): use rich_click for nicer help

### PR DESCRIPTION
**Pull request by efiop**
We could switch to typer, but for now going with least resistance and giving https://github.com/ewels/rich-click a try.

We already use rich in multiple places, so this is just making ui a little bit more coherent.

Screenshot 2024-03-21 at 02 51 09

Screenshot 2024-03-21 at 02 52 43

### CODE REVIEW

**Review by squat — APPROVED** [review]
Sounds cool! Does merging this change any functionality?

**Comment by squat** [comment]
btw, I don't love linking issues in private issue trackers to OSS projects/PRs since community has no idea what evil plans are being implemented. Maybe we can track these as basic GitHub issues

**Comment by efiop** [comment]
> Sounds cool! Does merging this change any functionality?

None that I'm aware of. Good point though, let me switch `feat` to something else...

> btw, I don't love linking issues in private issue trackers to OSS projects/PRs since community has no idea what evil plans are being implemented. Maybe we can track these as basic GitHub issues

Agreed. Just getting used to linear thing, so overusing it a bit too much right now. Will be going with normal gh issues here from now on.

File: projects/fal/pyproject.toml [hunk 1]
```diff
@@ -24,6 +24,7 @@ grpc-interceptor = "^0.15.0"
 colorama = "^0.4.6"
 portalocker = "^2.7.0"
 rich = "^13.3.2"
+rich_click = "*"
```
Comment (mederka) [inline projects/fal/pyproject.toml:27]: This didn't affect poetry.lock? Did we already have it there?

Comment (efiop) [reply projects/fal/pyproject.toml:27]: @mederka Sorry, that's on me. Will update.

Comment (efiop) [reply projects/fal/pyproject.toml:27]: @mederka Thanks again for the heads up. Added some hooks https://github.com/fal-ai/fal/pull/137

**Pull request merged**